tests/system/Filters/DebugToolbarTest.php

```diff
@@ -23,6 +23,16 @@ protected function setUp(): void
 
 	//--------------------------------------------------------------------
 
+	public function testDebugToolbarFilterExcept()
+	{
+		$data     = [
+			'testString' => 'bar',
+			'bar'        => 'baz',
+		];
+		$expected = '<h1>bar</h1>';
+		$this->assertEquals($expected, view('\Tests\Support\View\Views\simple', $data, []));
```

Comment:
This test won't actually run it through the controller and fire the filters themselves, so doesn't seem to test the new stuff you've added.